tests/vector/test_spaces.py

```diff
@@ -129,3 +133,78 @@ def test_iterate_custom_space(space, batch_space):
     for i, item in enumerate(iterator):
         assert item in space
     assert i == 3
+
+
+seeded_spaces = [
+    CustomSpace(seed=123),
+    Box(0, 10, (), seed=123),
+    Tuple([Box(0, 5, (), seed=123), Box(0, 3, (), seed=123)], seed=123),
+    Dict(
+        {"space-1": Box(0, 5, (), seed=123), "space-2": Box(0, 10, (), seed=123)},
+        seed=123,
+    ),
+    Discrete(5, seed=123),
+    MultiDiscrete([5, 3], seed=123),
+]
+
+
+@pytest.mark.parametrize(
+    "space", seeded_spaces, ids=[space.__class__.__name__ for space in seeded_spaces]
```

Comment:
Watch out, the seeded_spaces are shared between tests, and so their rng state is affected!
(BTW this hasn't caused an issue here, because you're only using `seeded_spaces` in one place, and your tests aren't dependant on the precise rng spate of the `space` they get from `seeded_spaces`.
Just thought I'd point this out, in case it's useful to you :)